src/components/TopBlocks.jsx

```diff
@@ -49,13 +48,15 @@ class BlockItem extends Component {
 }
 
 TopBlocks.propTypes = {
-  blockList: PropTypes.array
+  blockList: PropTypes.array,
+  getRpcServer: PropTypes.func
```

Comment:
What is the purpose line 52 here? The RPC server should already be fed by the `blockList` prop.